docs/DafnyRef/UserGuide.md

```diff
@@ -617,9 +618,15 @@ enforce-determinism = true
 warn-shadowing = true
 ```
 
-Under the section `[options]`, any options from the Dafny CLI can be specified using the option's name without the `--` prefix. When executing a `dafny` command using a project file, any options specified in the file that can be applied to the command, will be. Options that can't be applied or are misspelled, are ignored.
-
-When using a Dafny IDE based on the `dafny server` command, the IDE will search for project files by traversing up the file tree looking for the closest `dfyconfig.toml` file it can find. Options from the project file will override options passed to `dafny server`.
+- At most one `.toml` file may be named on the command-line; when using the command-line no `.toml` file is used by default.
+- `.toml` files with syntax errors will cause the invocation of dafny to fail (with a preprocessing error).
```

Comment:
This is so much in line with my expectations that I don't think it's useful to document.

Reply:
Its a reference manual, not a tutorial -- so my expectations are that the document defines the behavior.
Reasonable alternate behavior is that later things on the command-line supersede earlier ones, as options do.

Reply:
I'll omit line 622 though. (I have encountered tools that silently omit badly formatted input files, though I don't remember where at the moment).